example/integrations/tracker-red-box-ltd.tsx

```diff
@@ -34,63 +25,55 @@ const Icon: React.FC = () => (
   </svg>
 )
 
-// ensure that the tracker script will be initialized once in runtime
-let wasInitialized = false
+const ScriptInjector: React.FC = () => {
```

Comment:
this integration injects a script tag and manipulates window as many tracking scripts do